packages/react-checkbox/src/components/Checkbox/useCheckbox.tsx

```diff
@@ -49,7 +48,7 @@ export const useCheckbox_unstable = (props: CheckboxProps, ref: React.Ref<HTMLIn
   let checkmarkIcon;
   if (mixed) {
     if (shape === 'circular') {
-      checkmarkIcon = size === 'large' ? <Circle16Filled /> : <Circle12Filled />;
+      checkmarkIcon = <CircleFilled />;
```

Comment:
Should a similar change be made for the square icon below?

Reply:
I tried that, but the resizable version is fuzzy at 12px. I decided to leave it as-is to keep the crisp icon.